src/Configuration/Form/Checkbox.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Configuration\Form;
+
+use MyParcelCom\Integration\Configuration\Field;
+use MyParcelCom\Integration\Configuration\Properties\JsonSchemaProperty;
+use MyParcelCom\Integration\Configuration\Properties\PropertyType;
+
+class Checkbox implements Field
+{
+    public function __construct(
+        public readonly string $name,
+        public readonly string $description,
+        public readonly bool $isRequired = false,
+        public readonly ?string $hint = null,
+    ) {
+    }
+
+    public function isRequired(): bool
+    {
+        return $this->isRequired;
+    }
```

Comment:
![image](https://github.com/user-attachments/assets/d062dc75-4eb1-4778-a85f-662da0ce0251)

P.S. I'll just remove it from here 
```suggestion
```